lib/proxy/plugin_validators.rb

```diff
@@ -7,6 +7,7 @@ def initialize(plugin, setting_name)
 
     def validate!
       setting_value = @plugin.settings.send(@setting_name)
+      return true if setting_value.nil?
```

Comment:
Only chef plugin is going to have an issue with this approach due to chef_ssl_pem_file setting (https://github.com/theforeman/smart_proxy_chef/blob/master/lib/smart_proxy_chef_plugin/chef_plugin.rb#L10). I can't see it being used anywhere though -- the fix is going to be trivial.